Misc/NEWS.d/next/Core and Builtins/2020-03-07-12-35-31.bpo-39890.zr6CNx.rst

```diff
@@ -0,0 +1 @@
+Don't mutate the AST when compiling starred assignments.
```

Comment:
This is an internal change without user impact so we don't need NEWS entry for it.

Reply:
Done! @pablogsal